ft/src/test/resources/projects/octopus-releng-plugin-compatibility/gradle.properties

```diff
@@ -1 +1 @@
-octopus-release-management.version=2.0.27
\ No newline at end of file
+octopus-release-management.version=1.0
```

Comment:
keep the value empty in order to push developers to manually specify latest version of RM gradle plugin